tools/glyph-studio/py/section_propagate_eval.py

```
@@ -0,0 +1,73 @@
+
+def load_words(snapshot: str, section_map: dict):
+    import chromadb
+
+    client = chromadb.PersistentClient(path=snapshot)
+    coll = [c for c in client.list_collections() if c.name == "words"][0]
+    n = coll.count()
+    emb, labels, receipts = [], [], []
+    B = 5000
+    for off in range(0, n, B):
+        r = coll.get(limit=B, offset=off, include=["metadatas", "embeddings"])
+        for md, e in zip(r["metadatas"], r["embeddings"]):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)